gnovm/pkg/gnolang/types.go

```diff
@@ -2437,7 +2446,7 @@ func specifyType(store Store, lookup map[Name]Type, tmpl Type, spec Type, specTy
 					return // ok
 				} else {
 					if isUntyped(spec) {
```

Comment:
is there any chance `spec` can be untypedBigInt ?

Reply:
I'm not sure. Maybe. Why do you ask?